edx_proctoring/migrations/0013_proctoredexamsoftwaresecurereview_attempt_code_constraint.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.17 on 2021-01-12 13:55
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('edx_proctoring', '0012_proctoredexamstudentattempt_time_remaining_seconds'),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
I noticed this just now. This is going to cause production issue during deployment. See https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Howtorenameacolumn